lib/evmone/eof.hpp

```diff
@@ -61,4 +106,19 @@ enum class EOFValidationError
 
 /// Returns the error message corresponding to an error code.
 [[nodiscard]] EVMC_EXPORT std::string_view get_error_message(EOFValidationError err) noexcept;
+
+/// Loads big endian int16_t from data. Unsafe.
+/// TODO: Move it to intx
+inline int16_t read16bes(const uint8_t* data) noexcept
```

Comment:
Copying from https://github.com/ethereum/evmone/pull/563/files#r1124926925:
- Would make this take an iterator as it makes the call site nicer.
- Would rename these to `read_uint16_be` and `read_int16_be`.